lib/browser.ts

```diff
@@ -365,17 +364,21 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
         ng12Hybrid_ = ng12Hybrid;
       }
     });
-    this.driver.getCapabilities().then((caps: Capabilities) => {
-      // Internet Explorer does not accept data URLs, which are the default
-      // reset URL for Protractor.
-      // Safari accepts data urls, but SafariDriver fails after one is used.
-      // PhantomJS produces a "Detected a page unload event" if we use data urls
-      let browserName = caps.get('browserName');
-      if (browserName === 'internet explorer' || browserName === 'safari' ||
-          browserName === 'phantomjs' || browserName === 'MicrosoftEdge') {
-        this.resetUrl = 'about:blank';
-      }
-    });
+    this.ready = this.driver.controlFlow()
```

Comment:
`this.driver.getSession()` is not necessarily a promise which is on the control flow, so this is needed to ensure that the other stuff we stick onto `browser.ready` is on the control flow